packages/web_benchmarks/lib/src/compilation_options.dart

```diff
@@ -7,11 +7,15 @@
 /// This object holds metadata that is used to determine how the benchmark app
 /// should be built.
 class CompilationOptions {
-  /// Creates a [CompilationOptions] object.
-  const CompilationOptions({
```

Comment:
Removing a constructor from a public class is a breaking change, and all of our packages follow semver. If you want to remove this, the package needs a major version bump.